R/tidyverse.R

```diff
@@ -101,6 +104,47 @@ use_tidy_description <- function() {
   invisible(TRUE)
 }
 
+#' @export
+#' @rdname tidyverse
+use_tidy_dependencies <- function() {
+  if (!check_has_package_doc()) {
+    return(ui_stop("Requires package docs"))
+  }
+
+  use_dependency("rlang", "Imports")
+  use_dependency("ellipsis", "Imports")
+  use_dependency("lifecycle", "Imports")
+  use_dependency("cli", "Imports")
+  use_dependency("glue", "Imports")
+  use_dependency("withr", "Imports")
+
+
+  # standard imports
+  imports <- any(
+    roxygen_ns_append("@import rlang"),
+    roxygen_ns_append("@importFrom glue glue"),
+    roxygen_ns_append("@importFrom lifecycle deprecated")
```

Comment:
Is there a reason we wouldn't import all of lifecycle?

Reply:
The reason for importing `deprecated()` is that you use it in arguments; for everything else I think it's better to be explicit.